packages/remix/src/performance/client.tsx

```diff
@@ -135,5 +137,5 @@ export function withSentryRouteTracing<P extends Record<string, unknown>, R exte
 
   // @ts-ignore Setting more specific React Component typing for `R` generic above
   // will break advanced type inference done by react router params
-  return SentryRoot;
+  return withErrorBoundary(SentryRoot);
```

Comment:
We’ll need to make sure users can supply errorboundary options, so things like fallback can be configured. We can pass it in as an option?

Reply:
One thing to note is what we do if someone doesn’t want an error boundary here.